index.js

```diff
@@ -139,9 +193,12 @@ const sortWorkspaces = (workspaces) => {
     sortedWorkspaces.packages = uniqAndSortArray(workspaces.packages)
   }
 
-  // Then add catalog if it exists and sort it like dependencies
+  // Then add catalog if it exists and sort it using locale-aware comparison
+  // (npm-style)
   if (workspaces.catalog) {
-    sortedWorkspaces.catalog = sortDependenciesLikeNpm(workspaces.catalog)
+    sortedWorkspaces.catalog = sortObjectBy((a, b) => a.localeCompare(b, 'en'))(
+      workspaces.catalog,
+    )
```

Comment:
I forgot what's this, should it alway sort like NPM? If so, please restore to use `sortDependenciesLikeNpm`.

Reply:
Workspaces are also supported by yarn IIRC, so this should probably use `sortDependencies` also.

https://github.com/keithamus/sort-package-json/blob/main/tests/workspaces.js

> ```js
>       catalog: {
>         'is-even': 'npm:is-even@1.0.0',
>         'is-odd': 'npm:is-odd@3.0.1',
>         'alpha-lib': 'npm:alpha-lib@1.0.0',
>       },
> ```